local CircylarSelector = require(game.ReplicatedStorage.Shared.Modules.CircularSelector)
local ClientConfig = require(script.Parent.Configs.ClientConfig)
-- ToDo: добавить ивенты и функции для взаимодействия с сервером
-- скривать и закрывать кнопку конфирм между раундами
-- отображать прогресс бар между раундами убирая не нужные картинки
-- отображать победителя раунда и очки
-- отображать очки игроков в лидерборде
-- отображать победителя игры
-- анимации при победе и поражении
-- звуки при победе и поражении
-- анимации для прогресс бара
-- звуки для прогресс бара
-- анимации для кнопки конфирм
-- доработать модуль по UIAnimations для всех этих анимаций
